keras_core/utils/image_dataset_utils.py

```diff
@@ -180,24 +185,6 @@ def image_dataset_from_directory(
             f"Received: color_mode={color_mode}"
         )
 
-    interpolation = interpolation.lower()
```

Comment:
This block should not be removed.

Reply:
Done, Added it back again but since not all interpolation methods are supported in all backend types and if user uses `smart_resize()` then `backend.resize()` is called with current backend then already ValueError have been raised specific to backend.